app/AcquireIntersonArrayBMode.cxx

```diff
@@ -147,13 +147,18 @@ int main( int argc, char * argv[] )
     std::cerr << "Invalid requested depth for probe." << std::endl;
     }
 
+  const double ns = ContainerType::MAX_SAMPLES; // number of samples
+  const double fs = frequencies[ frequencyIndex ] / 1000; // [kHz]=[samples/ms] - frequency per sample
+  double depth = c * ns / ( 2 * fs );
+
   const int scanWidth = container.GetWidthScan();
   const int scanHeight = container.GetHeightScan();
   std::cout << "Width = " << width << std::endl;
   std::cout << "Height = " << height << std::endl;
   std::cout << "ScanWidth = " << scanWidth << std::endl;
   std::cout << "ScanHeight = " << scanHeight << std::endl;
   std::cout << "MM per Pixel = " << container.GetMmPerPixel() << std::endl;
```

Comment:
Ah, makes sense.

Since (I assume) the RF output is not using the scan converter, then these changes should be made in `AcquireIntersonArrayRF` and not in `AcquireIntersonArrayBMode`.

So `imageSize[0]` is always a fixed 512, or does this change with the the depth setting? Is `container.MAX_SAMPLES` unused?

We should use `container.GetMmPerPixel()` for `imageSpacing[0]`? And for `imageSpacing[1]`, or is that fixed?